tasks/src/document-question-answering/data.ts

```diff
@@ -0,0 +1,57 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "Dataset from the 2020 DocVQA challenge. The documents are taken from the UCSF Industry Documents Library.",
```

Comment:
Do we have a canonical dataset that the code is implemented within `datasets` library on Hub? @ankrgyl

Reply:
I think @NielsRogge's https://huggingface.co/datasets/nielsr/docvqa_1200_examples_donut/viewer/nielsr--docvqa_1200_examples_donut/train can be used to at least partially train Donut. I don't think DocVQA is fully uploaded on the hub. DocQuery itself is trained over DocVQA and a "document" version of SQuAD that we created but have not published on the hub.

Reply:
@merveenoyan a [registration](https://rrc.cvc.uab.es/?ch=17&com=downloads) is required to download the DocVQA data. I figured that it might be better to have a dataset page that references the useful links, although it is not ideal 😇